### PR TITLE
mail-filter/milter-regex: Replacement for outdated ebuild

### PR DESCRIPTION
**Pull request by rseichter**
This is a replacement for the milter-regex 1.9 ebuild. Version 1.9 was released seven years ago, and the old ebuild was based on EAPI 4. The new ebuild for milter-regex 2.2 (released a few days ago) uses EAPI 7.

Bug: https://bugs.gentoo.org/666758
Package-Manager: Portage-2.3.49, Repoman-2.3.10
Signed-off-by: Ralph Seichter <gentoo@seichter.de>

### CODE REVIEW

File: mail-filter/milter-regex/milter-regex-2.2.ebuild [hunk 1]
```diff
@@ -0,0 +1,56 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit toolchain-funcs
+inherit user
```
Comment (Zlogene) [inline mail-filter/milter-regex/milter-regex-2.2.ebuild]: single inherit please

Comment (rseichter) [reply mail-filter/milter-regex/milter-regex-2.2.ebuild]: Fixed 'inherit' statement.

**Comment by rseichter** [comment]
@Zlogene : I made the change you asked for. Is there anything else required?

**Comment by Polynomial-C** [comment]
Hi Palph,

sorry that this PR didn't get any more attention. Obviously our net-mail team is kind of defunct I'm afraid.
After looking at your changes, I have to raise some concerns that should be addressed before your PR gets merged. Please understand that despite my comment here, I would prefer to not do the final merge of this PR, simply because I neither belong to net-mail team nor did I ever use this package.

* Please do not replace the existing ebuild even though it's hopelessly outdated and might even be broken. The usual Gentoo way of updating a package is to add a new revision/version of a package, reset all KEYWORDS to ~arch (unstable) and let the new ebuild sit in ~arch for at least one month before it can be stabilized and replace the old ebuild.
* Please do not replace/patch the init script in FILESDIR but rather add a new init-script file to FILESDIR and let the ebuild install that new file via `doinitd` or `newinitd` helper functions.
* In your new ebuild, the whole `src_prepare()` function can be replaced by a single line: 
`PATCHES=( "${FILESDIR}/${P}-gentoo.patch" )`

**Comment by rseichter** [comment]
Thank you for looking into this, Lars.

> Obviously our net-mail team is kind of defunct I'm afraid.

The term "suboptimal" comes to mind. ☹️

> After looking at your changes, I have to raise some concerns that should be addressed before your PR gets merged.

I made the changes you requested.

> Please understand that despite my comment here, I would prefer to not do the final merge of this PR, simply because I neither belong to net-mail team nor did I ever use this package.

Do you have any suggestion as to who would be the right person to merge this pull request, and how best to contact her/him? Is there any fallback structure among the Gentoo developer teams?

**Comment by Polynomial-C** [comment]
> Do you have any suggestion as to who would be the right person to merge this pull request, and how best to contact her/him? Is there any fallback structure among the Gentoo developer teams?

Well, since you added yourself as maintainer of this package, I suggest to get in direct contact with one of our [proxy maintainers](https://wiki.gentoo.org/wiki/Project:Proxy_Maintainers), explaining the issue to him and ask for merge.
There's also the chance to find someone in freenode IRC `#gentoo-proxy-maint` channel.

File: mail-filter/milter-regex/metadata.xml [hunk 2]
```diff
@@ -1,10 +1,20 @@
...
+		<email>net-mail@gentoo.org</email>
+		<name>Net-Mail</name>
+	</maintainer>
+	<longdescription>
+		milter-regex is a very simple milter-based plugin that rejects or
+		discards messages matching regular expressions. It doesn't add much
+		processing overhead, so even a busy mail server can afford to run it.
+	</longdescription>
+	<upstream>
+		<doc>http://www.benzedrine.ch/milter-regex.html</doc>
```
Comment (candrews) [inline mail-filter/milter-regex/metadata.xml]: Please use `https`

File: mail-filter/milter-regex/milter-regex-2.2.ebuild [hunk 3]
```diff
@@ -0,0 +1,51 @@
+# Copyright 1999-2018 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit toolchain-funcs user
+
+DESCRIPTION="A milter-based regular expression filter"
+HOMEPAGE="http://www.benzedrine.ch/milter-regex.html"
```
Comment (candrews) [inline mail-filter/milter-regex/milter-regex-2.2.ebuild]: `https` please

File: mail-filter/milter-regex/milter-regex-2.2.ebuild [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+SRC_URI="http://www.benzedrine.ch/${P}.tar.gz"
```
Comment (candrews) [inline mail-filter/milter-regex/milter-regex-2.2.ebuild]: and `https` here too

**Comment by rseichter** [comment]
> Can you please add the proxy maint info to metadata.xml?

Done. I also changed URLs from http to https.

**Comment by candrews** [comment]
a199b66a6767c285587d9a22e06a42078e8a5684

**Pull request closed without merging**